### PR TITLE
[IE TESTS] Behavior2.0 -> ExecutableNetwork

### PR DESCRIPTION
**Pull request by iefode**
### Details:
image

### Tickets:
 - *63209*

### CODE REVIEW

File: inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/keep_assign.cpp [hunk 1]
```diff
@@ -32,10 +34,10 @@ TEST_P(ExecGraphKeepAssignNode, KeepAssignNode) {
 
     // Some simple graph with Memory(Assign) node                     //    in   read     //
     auto input = make_shared<Parameter>(type, shape);                 //    | \  /        //
-    auto mem_i = make_shared<Constant>(type, shape, 0);               //    |  mul        //
-    auto mem_r = make_shared<ReadValue>(mem_i, "id");                 //    | /  \        //
+    auto mem_i = make_shared<Constant>(type, shape, 0);          //    |  mul        //
```
Comment (ilyachur) [inline inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/keep_assign.cpp:37]: Incorrect indentation

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/executable_network/unique_node_names.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        ASSERT_NE(rtInfo.end(), it);
+        auto opType = std::dynamic_pointer_cast<ngraph::VariantImpl<std::string>>(it->second);
+        ASSERT_NE(nullptr, opType);
+
+        if (opType->get() == "Reorder") {
+            numReorders++;
+        }
+    }
+
+    ASSERT_EQ(numReorders, expectedReorders) << "Expected reorders: " << expectedReorders << ", actual reorders: " << numReorders;
```
Comment (ilyachur) [inline inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/executable_network/unique_node_names.cpp:42]: Looks like it is not a purpose of the test, name doesn't have any information about this plugin specific check.

Comment (iefode) [reply inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/executable_network/unique_node_names.cpp:42]: Thanks:)

**Comment by iefode** [comment]
Close according to https://github.com/openvinotoolkit/openvino/pull/7579

**Pull request closed without merging**